docusaurus/docs/learn/core-concepts/security/authentication/50-external-id-claims.md

```diff
@@ -1,4 +1,6 @@
-# External JWT Signers
+# External ID Claims
```

Comment:
I wouldn't suggest introducing a new term, "External ID Claim" without defining what it is. That term doesn't appear anywhere I know of other than here (i.e. not in APIs, CLIs, or code).

It seems like you mean to cover both External JWT Signers and X509 Claims here. If you are going to do that I suggest a list at the top explaining the new term and the two things described below so that the reader has a framework to start out with.

Reply:
Good callout. That's what I meant. It makes more sense to document the types of claims in this article, and types of enrollments in the enrollments article, because claims and enrollments are fundamentally different ways of authenticating an Edge Identity. A brief, user-centric comparison might be helpful too, e.g., "Enrollments are typically used for long-lived credentials, and claims are well-suited for short-lived credentials."